services/user.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User

```

Comment:
The import of `User` from `db.models` is unnecessary since `get_user_model()` is used throughout the file. Consider removing this import to clean up the code.